wrappers/hnysqlx/sqlx.go

```diff
@@ -75,7 +74,7 @@ func (db *DB) Beginx() (*Tx, error) {
 
 func (db *DB) BeginTxx(ctx context.Context, opts *sql.TxOptions) (*Tx, error) {
 	var err error
-	ev, sender := internal.BuildDBEvent(ctx, db.Builder, "")
+	ctx, span, sender := common.BuildDBSpan(ctx, db.Builder, "")
```

Comment:
hrmph. well I suppose this makes sense. It'll get a bit tricky to figure out sampling if you pass in an empty context to this stuff but I suppose that's something to figure out another time. I agree that it's unexpected that passing an empty context would result in no events at all rather than just trace-less events.